Doc/library/stdtypes.rst

```diff
@@ -1687,8 +1687,8 @@ expression support in the :mod:`re` module).
    Return true if the string is a valid identifier according to the language
    definition, section :ref:`identifiers`.
 
-   Use :func:`keyword.iskeyword` to test for reserved identifiers such as
-   :keyword:`def` and :keyword:`class`.
+   Call :func:`keyword.iskeyword` to test whether string ``s`` is a reserved
+   identifier, such as :keyword:`def` and :keyword:`class`.
```

Comment:
Adding an example would clarify the usage and differences:

Keywords, which are a small subset of identifiers used as reserved words, are not ordinary identifiers. To test if a string is a keyword, call :func:`keyword.iskeyword` to test whether a string ``s`` is a reserved identifier, such as :keyword: `def` and :keyword:`class`. 

An example shows their usage:

```
>>> import keyword

>>> string_ordinary = 'hello'
>>> string_keyword = 'def'

>>> string_ordinary.isidentifier()
True
>>> keyword.iskeyword(string_ordinary)
False
>>> string_keyword.isidentifier()
True
>>> keyword.iskeyword(string_keyword)
True
```

Reply:
This proposed change is in an entry about identifiers, not keywords.  Expanding the keyword module doc would be a separate discussion.  If we did something, I would it a bit shorter, such as
```
>>> from keyword import iskeyword

>>> 'hello'.isidentifier(), iskeyword('hello')
True, False
>>> 'def'.isidentifier(), iskeyword('def')
True, True
```

Reply:
I like @terryjreedy’s suggested example. More compact but captures the importance of contrasting both uses.